app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1659,6 +1688,23 @@ BrowserWorld::CreateSkyBox(const std::string& aBasePath, const std::string& aExt
   }
 }
 
+void BrowserWorld::CreateEnvironment() {
+  ASSERT_ON_RENDER_THREAD();
+  m.rootEnvironment = Transform::Create(m.create);
+  m.rootEnvironment->AddLight(Light::Create(m.create));
+
+  vrb::TransformPtr model = Transform::Create(m.create);
```

Comment:
We are setting this transform to rootEnvironment: `m.rootEnvironment->SetTransform(m.device->GetReorientTransform());`

I guess it will depend on how we include the env models in the future? will the FxR world orientation transform be included in the GLTF or could we need extra transform (as it was required for the space platform)?

Reply:
My hope is that they will have the correct orientation. I think we can look into supporting a custom transform on a case by case basis? It's not a big deal I guess I was just curious if was need. Ideally I would like to not have any transform over the static terrain, but that might be unrealistic.